### ISSUE TITLE
Bootstrap not included in create/delete/details/edit/index web pages

### ISSUE THREAD

**Issue by Albert-Gorup** [dotnet/AspNetCore.Docs#18498]
My create/delete/details/edit/index web pages don't include bootstrap so they look quite different form the picture on this page. Did I have to do something, such as include a layout page?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6719f08e-3bd7-dc1a-71df-f2ef9fbca9d8
* Version Independent ID: 7096fdb3-612e-9e00-bd0b-8ea4886a09ce
* Content: [Add a model to a Razor Pages app in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/tutorials/razor-pages/model?view=aspnetcore-3.1&tabs=visual-studio)
* Content Source: [aspnetcore/tutorials/razor-pages/model.md](https://github.com/dotnet/AspNetCore.Docs/blob/master/aspnetcore/tutorials/razor-pages/model.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-tutorials**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by Rick-Anderson** [comment]
Look in wwwroot\lib\bootstrap\dist\css\bootstrap.css

**Comment by Albert-Gorup** [comment]
Allow me to re-phrase.
My web page doesn't look like the example provided. I looked at the web page and saw this:


<h1>Index</h1>

<p>
    <a asp-page="Create">Create New</a>
</p>
<table class="table">
    <thead>
        <tr>
            <th>
                Title
            </th>
            <th>
                ReleaseDate
            </th>
            <th>
                Genre
            </th>
            <th>
                Price
            </th>
            <th></th>
        </tr>
    </thead>
    <tbody>
    </tbody>
</table>


In other words, the web page generated doesn't have all the other HTML codfe of the other pages. No doctype, no link to bootstrap.min.css, etc …
It doesn't look like the example you provided because it lacks the HTML to make it look like your example.

**Comment by Albert-Gorup** [comment]
![webpage1](https://user-images.githubusercontent.com/65995393/83223397-5ab4d200-a140-11ea-9765-dec68e64c119.jpg)
![webpage2](https://user-images.githubusercontent.com/65995393/83223410-60121c80-a140-11ea-97c7-3f9e50a7e7ac.jpg)

**Comment by Rick-Anderson** [comment]
The tutorial works. I'm not sure what step you missed. You might try asking on StackOverflow,.

**Comment by Rick-Anderson** [comment]
You're missing the layout.

**Comment by Albert-Gorup** [comment]
![webpage3](https://user-images.githubusercontent.com/65995393/83266473-7f349c80-a188-11ea-8781-8cb1197aae02.jpg)

Yes. I am missing the layout. I am also missing a Shared directory with two files: _Layout.cshtml and _ValidationScriptsPartial.cshtml

**Comment by Rick-Anderson** [comment]
I'd start over and compare to the completed app.

**Comment by Rick-Anderson** [comment]
The Index view is using the correct layout.

**Comment by wadepickett** [comment]
@Rick-Anderson, it is not clear to me where this one is at.  You reopened it so I'm thinking you found something that needs to be corrected, but it is not clear what that is.
